src/SFA.DAS.RoatpOversight.Web/Views/Oversight/Applications.cshtml

```diff
@@ -56,22 +56,41 @@
                                     <th scope="col" class="govuk-table__header">Application submitted date</th>
+                                    <th scope="col" class="govuk-table__header">Outcome</th>
                                 </tr>
                             </thead>
                             <tbody class="govuk-table__body">
                                 @foreach (var application in Model.ApplicationDetails)
                                 {
-                                    <tr class="govuk-table__row">
-                                        <td data-label="Organisation name" class="govuk-table__cell">
-                                            <a class="govuk-link govuk-!-margin-bottom-0" asp-action="Outcome" asp-route-id="@application.ApplicationId">
-                                                @application.OrganisationName
-                                            </a>
-                                        </td>
-                                        <td data-label="UKPRN" class="govuk-table__cell">@application.Ukprn</td>
-                                        <td data-label="Provider route" class="govuk-table__cell">@application.ProviderRoute</td>
-                                        <td data-label="Application reference number" class="govuk-table__cell">@application.ApplicationReferenceNumber</td>
-                                        <td data-label="Application submitted date" class="govuk-table__cell">@application.ApplicationSubmittedDate.ToSfaShortDateString()</td>
-                                    </tr>
+                                <tr class="govuk-table__row">
+                                    <td data-label="Organisation name" class="govuk-table__cell">
+                                        <a class="govuk-link govuk-!-margin-bottom-0" asp-action="Outcome" asp-route-id="@application.ApplicationId">
+                                            @application.OrganisationName
+                                        </a>
+                                    </td>
+                                    <td data-label="UKPRN" class="govuk-table__cell">@application.Ukprn</td>
+                                    <td data-label="Provider route" class="govuk-table__cell">@application.ProviderRoute</td>
+                                    <td data-label="Application reference number" class="govuk-table__cell">@application.ApplicationReferenceNumber</td>
+                                    <td data-label="Application submitted date" class="govuk-table__cell">@application.ApplicationSubmittedDate.ToSfaShortDateString()</td>
+                                    <td data-label="Outcome" class="govuk-table__cell">
+                                        @{
+                                            bool OutcomePass = application.GatewayReviewStatus == GatewayReviewStatus.Pass 
```

Comment:
Could we determine the pass/fail status inside a property on the viewmodel, and keep the markup more readable? It'd be unit testable that way, too.